mariadb/values.yaml

```diff
@@ -1,6 +1,20 @@
-imageTag: 10.1.14-r1
+# Bitnami MariaDB image version
+# ref: https://hub.docker.com/r/bitnami/mariadb/tags/
+imageTag: 10.1.14-r2
 
+# Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+imagePullPolicy:
```

Comment:
This is an small thing but in my opinion, when I see a key without a value it makes me think that I need to fill it out. I do not think the same when it is commented out.

I am wondering if we should:

* For non required keys (ones that have defaults) comment out the keys.
* For keys that need to be filled out to work, leave them uncommented.

What do you think?